terraform/cf/aws/security-groups.tf

```diff
@@ -185,3 +185,55 @@ resource "aws_security_group" "cf_router_lb_internal_security_group" {
     Environment = "${var.environment}"
   }
 }
+
+resource "aws_security_group" "cf_ssh_lb" {
+  name        = "${var.environment}_cf_ssh_lb"
+  description = "CF SSH traffic from load balancer"
+  vpc_id      = "${var.vpc_id}"
+
+  tags {
+    Name        = "${var.environment}-cf-ssh-lb-security-group"
+    Environment = "${var.environment}"
+  }
+}
+
+resource "aws_security_group_rule" "allow_tcp_2222_from_whitelist" {
+  security_group_id = "${aws_security_group.cf_ssh_lb.id}"
+  type              = "ingress"
+  protocol          = "tcp"
+  from_port         = 2222
+  to_port           = 2222
+  cidr_blocks       = ["${var.ingress_whitelist}"]
+  description       = "Provide egress SSH traffic"
```

Comment:
Should be `ingress` I assume?

Reply:
Perhaps "Allow SSH proxy traffic from whitelist"?